services/web/server/src/simcore_service_webserver/announcements/_models.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+from typing import Any, ClassVar, Literal
+
+import arrow
+from pydantic import BaseModel, validator
+
+
+# NOTE: this model is used for BOTH
+# - parse+validate from redis
+# - schema in the response
+class Announcement(BaseModel):
```

Comment:
This model is very nice!
I would spend a tiny bit more effort and have a simple CLI entry on the webserver from where we can push these announcements in redid. This way we know for sure they are correctly validated.

Nothing fancy, it can take a raw string containing the stringified json object

Reply:
I get your point but this is so **unimportant but urgent** that we accept this version to be manually added in the redis database. 
Nonetheless, i have added a follow up for an entrypoint to create that in https://github.com/ITISFoundation/osparc-issues/issues/1006